src/Nethermind/Nethermind.Runner/configs/kiln.cfg

```diff
@@ -60,5 +60,8 @@
   },
   "Mining": {
     "MinGasPrice": 0
+  },
+  "Pruning": {
+    "Mode": "None"
```

Comment:
So, kiln by default no longer have pruning?

Reply:
kiln.cfg is unfortunately archive (mi mistake with convention - after the merge kiln will be deprecated)
we have kiln_fastSync.cfg with pruning